test/unit/parseGitmodulesDependencies.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { expect } from 'chai';
+import mock from 'mock-fs';
+import { parseGitmodulesDependencies } from '../../src/parseGitmodulesDependencies';
+
+describe('parseGitmodulesDependencies', () => {
+  before(() => {
+    mock.restore();
+  });
+  after(() => {
+    mock.restore();
+  });
+  describe('with no gitmodules file', function () {
+    it('should return an empty dependency list', function () {
+      const parsedDependencies = parseGitmodulesDependencies();
+      expect(parsedDependencies).to.deep.eq({});
+    });
+  });
+  describe('with an empty file', function () {
+    before(() => {
+      mock({
+        '.gitmodules': '',
+      });
+    });
+    it('should return an empty dependency list', function () {
+      const parsedDependencies = parseGitmodulesDependencies();
+      expect(parsedDependencies).to.deep.eq({});
+    });
+  });
+  describe('with a file with several dependencies', function () {
```

Comment:
Maybe worth also having a case where the gitmodules exist but is not in the format we expect it

Reply:
added them in 52b257b69b6b0c577fa73198e9bf6289551bde50